examples/cpp/seerep_ros_communication/src/hdf5dump.cpp

```diff
@@ -35,9 +36,14 @@ void DumpSensorMsgs::dump(const sensor_msgs::PointCloud2::ConstPtr& msg) const
 void DumpSensorMsgs::dump(const sensor_msgs::Image::ConstPtr& msg) const
 {
   boost::uuids::uuid uuid = boost::uuids::random_generator()();
+  std::string uuidString = boost::lexical_cast<std::string>(uuid);
   try
   {
-    m_ioImage->writeImage(boost::lexical_cast<std::string>(uuid), seerep_ros_conversions_pb::toProto(*msg));
+    m_ioImage->writeImage(uuidString, seerep_ros_conversions_pb::toProto(*msg));
+
+    // fill the vectors with labels if needed
+    //  std::vector<std::string> labelsAsStdVector, instancesAsStdVector;
+    //  m_ioImageCore->writeLabelsGeneral(uuidString, labelsAsStdVector, instancesAsStdVector);
```

Comment:
Why is the new feature commented out? :sweat_smile:

Reply:
that was just ment as a blueprint. Added the vectors as member vars and filled them with dummy data now.